sequencers/based/README.md

```diff
@@ -0,0 +1,90 @@
+
+- **Censorship Resistance**: Users can submit transactions directly to DA, bypassing the sequencer
+- **Decentralization**: No single entity controls transaction ordering
+- **Liveness**: The rollup continues operating as long as the DA layer is available
+- **Trustless**: Users don't need to trust the sequencer to include their transactions
+
+## Architecture
+
+```
+┌─────────────────────────────────────────────────────────────┐
+│                     Based Sequencer                          │
+│                                                               │
+│  ┌────────────────┐         ┌─────────────────┐            │
+│  │  Transaction   │         │   DA Retriever  │            │
+│  │     Queue      │◄────────│   (Interface)   │            │
+│  └────────────────┘         └─────────────────┘            │
+│         │                            │                       │
+│         │                            │                       │
+│         ▼                            ▼                       │
+│  ┌────────────────┐         ┌─────────────────┐            │
+│  │  GetNextBatch  │         │  Fetch Forced   │            │
+│  │    (Method)    │────────►│  Inclusion Txs  │            │
+│  └────────────────┘         └─────────────────┘            │
+│                                      │                       │
+└──────────────────────────────────────┼───────────────────────┘
+                                       │
+                                       ▼
+                              ┌─────────────────┐
+                              │   DA Layer      │
+                              │  (Forced Inc.   │
+                              │   Namespace)    │
+                              └─────────────────┘
+```
+
+## Features
+
+- **DA-Only Transaction Source**: Fetches transactions exclusively from DA forced inclusion namespace
+- **Batch Size Management**: Respects MaxBytes limits when creating batches
+- **Transaction Queue**: Buffers transactions when they exceed batch size limits
+- **DA Height Tracking**: Maintains synchronization with DA layer height
+- **Concurrent-Safe**: Thread-safe operations with mutex protection
+- **Automatic Height Management**: Handles "height from future" errors gracefully
+
+## Interface Compliance
+
+The Based Sequencer implements the `core/sequencer.Sequencer` interface:
+
+```go
+type Sequencer interface {
+    SubmitBatchTxs(ctx context.Context, req SubmitBatchTxsRequest) (*SubmitBatchTxsResponse, error)
+    GetNextBatch(ctx context.Context, req GetNextBatchRequest) (*GetNextBatchResponse, error)
+    VerifyBatch(ctx context.Context, req VerifyBatchRequest) (*VerifyBatchResponse, error)
+}
+```
+
+## Configuration
+
+The based sequencer uses the following configuration from `config.Config`:
+
+- `DA.ForcedInclusionNamespace`: Namespace for forced inclusion transactions
+- `DA.ForcedInclusionDAEpoch`: Number of DA blocks to scan per fetch
```

Comment:
i believe this should be part of genesis as its part of the fork choice rule

Reply:
https://github.com/evstack/ev-node/pull/2797/commits/3c4132d481b677a3dc72498b3a0524e6629592a5